src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -77,6 +82,12 @@ class TelegramBotHandler extends AbstractProcessingHandler
      */
     private $disableNotification;
 
+    /**
+     * Truncate message longer than MAX_MESSAGE_LENGTH
+     * @var ?bool
```

Comment:
why allowing `null` to do the same than `false` ?

Reply:
Made by analogy with other parameters (`$disableWebPagePreview`, `$disableNotification`). I can fix if this is not the best decision

Reply:
Fixed